extensions/ql-vscode/src/queries-panel/query-discovery.ts

```diff
@@ -124,7 +124,7 @@ export class QueryDiscovery
       return undefined;
     }
 
-    const rootDirectory = new FileTreeDirectory<string>(fullPath, name);
+    const rootDirectory = new FileTreeDirectory<string>(fullPath, name, env);
```

Comment:
Realised something silly I'm doing here. We should be passing in `this.app.environment` here so we're using the `App` that's being passed in where possible.